helpers/DeployHelper.ts

```diff
@@ -203,27 +239,82 @@ export class DeployHelper {
     };
   }
 
+  async upgradeState2(
```

Comment:
do we need upgradeState2 if we will use openzeppelin upgrades?
If yes, we need to create similar method to deploy with strategy param.
If not - we need to remove it.

Reply:
let's go with openzeppelin upgrades, removed